user/useradm.go

```diff
@@ -471,7 +473,9 @@ func (ua *UserAdm) Verify(ctx context.Context, token *jwt.Token) error {
 
 	//check service-specific claims - iss
 	if token.Claims.Issuer != ua.config.Issuer {
-		return ErrUnauthorized
+		if _, ok := ua.config.AllowedIssuers[token.Claims.Issuer]; !ok {
+			return ErrUnauthorized
+		}
```

Comment:
nit: By ensuring the issuer is part of the `AllowedIssuers` hashmap, you can remove the outer if condition.

Reply:
okey. (the idea here was to leave the if from 2016 :)